Feature#11 frontend dashboard admin

MauricioNC opened the pull request:
# Description

 - New controller for the dashboard_admin view.
 - Deleted the 'wdm' gem from the Gemfile.
 - Added styles using sass for the dashboard_admin desgin.
 
Note:
The 'wdm' gem was deleted because all of we are using a Linux environment

  ## Screenshots
Mockup
![image](https://user-images.githubusercontent.com/42758875/149801057-dcb5ac41-78db-428e-aa0a-0116805c73e6.png)

View
![image](https://user-images.githubusercontent.com/42758875/149800840-863446f7-dc5d-42ea-bfb3-e50136c34e9a.png)

Commit:
Generate new Controller

- New dashboard_admin controller was generated

Co-authored-by: MauricioNC <mauricio_navarro0@ucol.mx>

Commit:
Created route to dashboard admin view

- Gemfile: deleted the wdm gem and re-run bundle install command
- It was created the route for the dashboard admin view and defined an
  admin method, this will search on our app the file with the same name
fo the method.

Co-authored-by: MauricioNC <mauricio_navarro0@ucol.mx>

Commit:
Merge branch 'master' of github.com:BC-MAY-21-ROR/kata-07-reloj-checador-k7-n21-t3 into Feature#11Frontend_dashboard_admin

Commit:
Method admin for dashboard_admin view modified

The route for the dashboard_admin view was changed from 'admin' to 'dashboard_admin'  because already
we have an admin.html.erb for the login so, the new route in our system to find the correct resource (dashboard_admin) now is:

'dashboard_admin#dashboard_admin'.

The route to access to the dashboard admin its still been
'localhost:3000/admin'

Co-authored-by: MauricioNC <mauricio_navarro0@ucol.mx>

Commit:
Dahsboard_admin finished

- dashboard_admin.html.erb: contains all the HMTL code, the skeleton
  for the view.

- dashboard_admin.scss: contains all the code styling for the
  dashboard_admin view.

Commit:
Changes merged from master branch

DerekAyala commented:
@kevinnio @YilKanda could you give us a code review ?

xFerchoVJ commented on app/assets/stylesheets/dashboard_admin.scss:
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the dashboard_admin controller here.
```
If you don't need this file, you can delete it

MauricioNC replied:
ok, on it

MauricioNC replied:
Done.

AngelBFdev commented:
@ccarruitero can you give us a code review?

YilKanda commented on app/views/dashboard_admin/dashboard_admin.html.erb:
```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Dashboard admin</title>
+</head>
+<body>
```
Why did you leave this code here? Remember that this part is already in the application.erb

MauricioNC replied:
Done.

YilKanda commented on app/views/dashboard_admin/dashboard_admin.html.erb:
```diff
@@ -0,0 +1,37 @@
+        <input type="button" value="Absence report by month">
+      </div>
+    </div>
+</body>
+</html>
```
Why did you leave this code here? Remember that this part is already in the application.erb

MauricioNC replied:
Done.

YilKanda commented on test/controllers/dashboard_admin_controller_test.rb:
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class DashboardAdminControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
+  #   assert true
+  # end
```
Code that you do not use, delete it.

MauricioNC replied:
Just delete the code?, I mean, can leave the file in blank? @YilKanda

YilKanda replied:
I am not referring to that, what I am saying is that the code that is not necessary and that for this reason is _commented_, it is better to eliminate it.

In GitHub only your functional code should appear, for this reason, the commented code is not necessary.

MauricioNC replied:
Ok, got it

MauricioNC replied:
Done.

YilKanda commented on app/helpers/dashboard_admin_helper.rb:
```diff
@@ -0,0 +1,2 @@
+module DashboardAdminHelper
+end
```
If you don't use it, remove it.

MauricioNC replied:
Done.

YilKanda commented on app/assets/stylesheets/dashboard_admin.scss:
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the dashboard_admin controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: https://sass-lang.com/
```
If you don't use it, remove it.

MauricioNC replied:
Done.

Commit:
File deleted

We delete the 'dashboard_admin.scc' file from 'app/assets/stylesheets/dashboard_admin.scss' because we are linking this file from other rute

Commit:
Changes made

- It was deleted the code commented into the dashboar_admin_controller_test.rb,
  we don't need the content within this file, so we delete it.
- It was deleted the dashboard_admin_helper.rb, we wont use this file in
  the future.

In the rest of the files, it was deleted the lines from '<!DOCTYPE html>' to '<body>' and the lines </body> and </html>, becuase already we have the application.erb, this file contains the basic structure.

DerekAyala commented:
@kevinnio @YilKanda @xFerchoVJ could you give us one more code review ?

YilKanda reviewed (approved):
LGTM!

xFerchoVJ reviewed (approved):
LGTM!
